README.md

```diff
@@ -36,15 +36,40 @@ Run `yarn lint` to run the linter, or run `yarn lint:fix` to run the linter and
 
 ### Release & Publishing
 
-The project follows the same release process as the other libraries in the MetaMask organization:
-
-1. Create a release branch
-   - For a typical release, this would be based on `main`
-   - To update an older maintained major version, base the release branch on the major version branch (e.g. `1.x`)
-2. Update the changelog
-3. Update version in package.json file (e.g. `yarn version --minor --no-git-tag-version`)
-4. Create a pull request targeting the base branch (e.g. `main` or `1.x`)
-5. Code review and QA
-6. Once approved, the PR is squashed & merged
-7. The commit on the base branch is tagged
-8. The tag can be published as needed
+The project follows the same release process as the other libraries in the MetaMask organization. The GitHub Actions [`action-create-release-pr`](https://github.com/MetaMask/action-create-release-pr) and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) are used to automate the release process; see those repositories for more information about how they work.
+
+1. Choose a release version.
+
+- We follow SemVer, so the set of changes being released dictates what the release version will be. Analyze the changes to see whether they include any breaking changes, new features, or deprecations. See [the SemVer specification](https://semver.org/) for more information.
+
+2. If this release is backporting changes onto a previous release, then ensure there is a major version branch for that version (e.g. `1.x` for a `v1` backport release).
+
+- The major version branch should be set to the most recent release with that major version. For example, when backporting a `v1.0.2` release, you'd want to ensure there was a `1.x` branch that was set to the `v1.0.1` tag.
+
+3. Trigger the `workflow_dispatch` event for the `Create Release Pull Request` action to create the release PR.
```

Comment:
```suggestion
3. Trigger the [`workflow_dispatch`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch) event [manually](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) for the `Create Release Pull Request` action to create the release PR.
```

Reply:
github has decent docs for how to use the workflow_dispatch we should link to:

![image](https://user-images.githubusercontent.com/364566/124812250-3701b500-df18-11eb-8c5f-4ea897a66b65.png)

Reply:
Nice, thanks!

Maybe best to omit my attempt at explaining this as well, and just refer to the docs.